processes/fulfill_url_manifest.py

```diff
@@ -0,0 +1,166 @@
+import json
+import os
+import copy
+import logging
+from botocore.exceptions import ClientError
+
+from .core import CoreProcess
+from datetime import datetime, timedelta, timezone
+from model import Link
+from logger import createLog
+
+logger = createLog(__name__)
+
+class Fulfill_Process(CoreProcess):
+
+    def __init__(self, *args):
+        super(Fulfill_Process, self).__init__(*args[:4])
+
+        self.fullImport = self.process == 'complete' 
+        self.startTimestamp = None
+
+        # Connect to database
```

Comment:
let's remove this comment - I think generateEngine and createSession are clear enough that we are connecting to the database.

Reply:
I agree